pkg/sql/logictest/testdata/logic_test/drop_type

```diff
@@ -449,7 +449,9 @@ CREATE TABLE t (k schema_to_drop.typ PRIMARY KEY);
 statement ok
 CREATE TABLE schema_to_drop.t (k schema_to_drop.typ PRIMARY KEY);
 
-statement error pgcode 0A000 unimplemented: cannot drop type "test.schema_to_drop.(_)?typ" because other objects \(\[test\.public\.t\]\) still depend on it
+skipif config local-legacy-schema-changer
+skipif config local-mixed-22.2-23.1
+statement error pgcode 42P10 column "k" is referenced by the primary key
```

Comment:
DROP TYPE ... CASCADE still chokes when the type is used by a column in a PK. To be fully postgres-compatible we should do a primary index swap and set the new PK to a new rowid column. I haven't bothered.